src/contexts/v2/projectContext.ts

```diff
@@ -10,8 +10,8 @@ export type V2ProjectContextType = {
   fundingCycleMetadata: V2FundingCycleMetadata | undefined
   fundingCycle: V2FundingCycle | undefined
   queuedFundingCycle: V2FundingCycle | undefined
-  distributionLimit: string | undefined
-  queuedDistributionLimit: string | undefined
+  distributionLimit: BigNumber | undefined
```

Comment:
Fixes. These contract reads return BigNumber's and not hooks.

Reply:
Good catch, my bad!